tools/generate_from_protos.sh

```diff
@@ -24,7 +24,7 @@ command -v ${protoc_binary} && command -v ${protoc_grpc_binary} || {
     exit 1
 }
 
-plugin_list="action calibration gimbal camera core info mission offboard param telemetry"
+plugin_list="action calibration gimbal camera core info mission offboard param telemetry geofence"
```

Comment:
```suggestion
plugin_list="action calibration geofence gimbal camera core info mission offboard param telemetry"
```

Reply:
Just ran style checker. I tried to keep styling the same - probably just indentation/editor specific things :wink: